core/docker/default/etc/jvm.config

```diff
@@ -12,3 +12,5 @@
 -XX:PerBytecodeRecompilationCutoff=10000
 -Djdk.attach.allowAttachSelf=true
 -Djdk.nio.maxCachedBufferSize=2000000
+-XX:+UnlockDiagnosticVMOptions
```

Comment:
need a comment here + TODO with reference to Trino issue to remove when it's enabled by default